src/routes/docs/products/auth/verification/+page.markdoc

```diff
@@ -0,0 +1,282 @@
+import Appwrite
+
+let client = Client()
+    .setEndpoint("https://cloud.appwrite.io/v1")
+    .setProject("<PROJECT_ID>")
+
+let account = Account(client)
+
+let response = try await account.updateVerification(
+    userId: "<USER_ID>",
+    secret: "<SECRET>"
+)
+```
+
+```client-android-kotlin
+import io.appwrite.Client
+import io.appwrite.services.Account
+
+val client = Client(context)
+    .setEndpoint("https://cloud.appwrite.io/v1")
+    .setProject("<PROJECT_ID>")
+
+val account = Account(client)
+
+val response = account.updateVerification(
+    userId = "<USER_ID>",
+    secret = "<SECRET>"
+)
+```
+{% /multicode %}
+
+# Phone verification {% #phone-verification %}
+
+To verify a phone number, first ensure the user has a phone number set on their account:
+
+{% multicode %}
+```client-web
+const response = await account.updatePhone(
+    '+12065550100',  // phone
+    'password' // password
+);
+```
+
+```client-flutter
+Future result = account.updatePhone(
+    phone: '+12065550100',
+    password: 'password'
+);
+
+result.then((response) {
+    print(response);
+}).catchError((error) {
+    print(error.response);
+});
+```
+
+```client-apple
+let response = try await account.updatePhone(
+    phone: "+12065550100",
+    password: "password"
+)
+```
+
+```client-android-kotlin
+val response = account.updatePhone(
+    phone = "+12065550100",
+    password = "password"
+)
+```
+{% /multicode %}
+
+Then initiate verification by calling `createPhoneVerification`:
+
+{% multicode %}
+```client-web
+const response = await account.createPhoneVerification();
+```
+
+```client-flutter
+Future result = account.createPhoneVerification();
+
+result.then((response) {
+    print(response);
+}).catchError((error) {
+    print(error.response);
+});
+```
+
+```client-apple
+let response = try await account.createPhoneVerification()
+```
+
+```client-android-kotlin
+val response = account.createPhoneVerification()
+```
+{% /multicode %}
+
+After the user receives the verification code, complete verification by calling `updatePhoneVerification`:
+
+{% multicode %}
+```client-web
+const response = await account.updatePhoneVerification(
+    '[USER_ID]',  // userId
+    '[SECRET]' // secret
+);
+```
+
+```client-flutter
+Future result = account.updatePhoneVerification(
+    userId: '[USER_ID]',
+    secret: '[SECRET]'
+);
+
+result.then((response) {
+    print(response);
+}).catchError((error) {
+    print(error.response);
+});
+```
+
+```client-apple
+let response = try await account.updatePhoneVerification(
+    userId: "[USER_ID]",
+    secret: "[SECRET]"
+)
+```
+
+```client-android-kotlin
+val response = account.updatePhoneVerification(
+    userId = "[USER_ID]",
+    secret = "[SECRET]"
+)
+```
+{% /multicode %}
+
+# Role-based access {% #role-based-access %}
+
+You can restrict resource access to verified users only using permissions through the `user([USER_ID], "verified")` role. This role is automatically assigned after successful verification.
```

Comment:
This would restrict for a single user. Let's also mention `users('verified')` to restrict access to all/any verified user.